.agents/skills/javascript-testing-expert/SKILL.md

```
@@ -0,0 +1,399 @@
+---
+name: javascript-testing-expert
+description: Expert-level JavaScript testing skill focused on writing high-quality tests that find bugs, serve as documentation, and prevent regressions. Advocates for property-based testing with fast-check and protects against indeterministic code in tests. Does not cover black-box e2e testing.
+---
+
+> **⚠️ Scope:** Testing functions and components, not black-box e2e.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)